Tutorial_Templates/tutorial_template_msb_v1.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<p>It looks like this is an example from the earthdata library so we may want to clarify that in your comment. As a tutorial creator I may look at this and think I need to include this but may not realize that I can only do this if importing the earthdata library. This is also something we have been continuing to use for many of the Openscapes-related events and is a bit more generic. Maybe we could also include a reference to this: https://nasa-openscapes.github.io/2021-Cloud-Hackathon/tutorials/04_NASA_Earthdata_Authentication.html </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>